katello_certs_tools/katello_ssl_tool.py

```diff
@@ -482,14 +515,16 @@ def genServerCert(password, d, verbosity=0):
     finally:
         chdir(cwd)
 
-    out = out_stream.read(); out_stream.close()
-    err = err_stream.read(); err_stream.close()
+    out = out_stream.read()
+    out_stream.close()
+    err = err_stream.read()
+    err_stream.close()
 
     if ret:
         # signature for a mistyped CA password
-        if string.find(err, "unable to load CA private key") != -1 \
-          and string.find(err, "error:0906A065:PEM routines:PEM_do_header:bad decrypt:pem_lib.c") != -1 \
-          and string.find(err, "error:06065064:digital envelope routines:EVP_DecryptFinal:bad decrypt:evp_enc.c") != -1:
+        if err.find("unable to load CA private key") != -1 \
```

Comment:
I'm pondering what is more readable, this or

`if "unable to load CA private key" in err`

what do you think?